internal/test/wsu/wsu_test.go

```diff
@@ -68,6 +68,13 @@ func (f *wsuFramework) Setup(vmCount int, credentials []*types.Credentials, skip
 		fmt.Errorf("framework setup failed: %v", err)
 		return err
 	}
+	if err := f.GetLatestWMCBRelease(); err != nil {
```

Comment:
Why do we need them in WSU?

Reply:
The WSU e2e tests runs WSU twice, once with a released WMCB binary and then with a WMCB binary built from source.